verify/src/main/java/com/twilio/verify/TwilioVerify.kt

```diff
@@ -139,6 +140,15 @@ interface TwilioVerify {
     error: (TwilioVerifyException) -> Unit
   )
 
+  /**
+   * Clears local data, it will delete factors and key pairs in this device.
+   * Note: Calling this method will not delete factors in **Verify Push API**, so you need to
+   * delete them from your backend to prevent invalid/deleted factors when getting factors for
+   * an identity.
+   * @param then Block to be called when the operation finishes
+   */
+  fun clearLocalData(then: () -> Unit)
```

Comment:
I'm not sure of this name, doesn't seem to be descriptive enough, what do you think about `deleteFactorsAndKeyPairsLocally`

Reply:
For now we saving factors and key pairs, but we could save more information in a future. This method will remove everything we have for Verify

Reply:
So maybe `clearLocalStorage` ?

Reply:
Done